save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/cosv/VulnerabilityExt.kt

```diff
@@ -23,4 +24,6 @@ data class VulnerabilityExt(
         description = metadata.details,
         relatedLink = cosv.getRelatedLink(),
     )
+
+    fun getAllParticipants() = saveContributors.plus(metadata.user)
```

Comment:
`metadata.user` should be already in `saveContributors`?

Reply:
Found some problems with it, will fix it a little bit later.